src/config/loadGitHubOrgs.test.ts

```diff
@@ -0,0 +1,44 @@
+// Magic incantation. I don't have time to get to the bottom of this, but for
+// some reason the following two lines allow the import machinery to function
+// as desired. Without this, loadGitHubOrgs is undefined in config/index.ts.
+import { DRY_RUN } from '.'; // random simple envvar
+DRY_RUN; // Yes, this is necessary. 🧙
+
+import { loadGitHubOrgs } from './loadGitHubOrgs';
+
+describe('loadGitHubOrgs', function () {
+  it('basically works', async function () {
+    const orgs = loadGitHubOrgs({
+      GH_ORGS_YML: 'test/github-orgs.good.yml',
+      GH_KEY_BLAH: '--------begin----end-----------',
+    });
+    expect(orgs.get('zer-ner').appAuth.appId).toEqual(53); // tests type conversion
+    expect(orgs.get('hurple').appAuth.appId).toEqual(42);
+  });
+
+  it('mixes in a private key from the environment', async function () {
+    const actual = loadGitHubOrgs({
+      GH_APP_PRIVATE_KEY_FOR_GETSENTRY: 'cheese',
+      GH_ORGS_YML: 'github-orgs.yml',
```

Comment:
is it worth also testing without passing in `GH_ORGS_YML`? Since it looks like that would be the way it's done in prod?

Reply:
Good idea, done in 885b332f9a4b1ca0943d7660d076a2d611c5c739.